app/zustand/global.ts

```diff
@@ -1,27 +1,68 @@
+import { Session, StringSession } from '@/vendor/telegram/sessions'
 import { SpaceDID } from '@storacha/ui-react'
 import { create } from 'zustand'
 import { persist } from 'zustand/middleware'
+import { Buffer } from "buffer/"
+const CURRENT_VERSION = "1";
 
 interface User {
 	id: number
 	name: string
 	email: string
 }
 
+const saveSessionToString = (session?:  Session | string ) => {
```

Comment:
one thought I had is since we're now only actually using this one the server, it might be nice to save this to the cloud backup instead so that people who use the app on multiple devices don't have to do the authorization twice. I actually implemented it here at one point: https://github.com/storacha/tg-miniapp/blob/feat/experiment-server/app/providers/telegram.tsx#L58 (this is an unmerged branch with various incompatibilites

Reply:
Woow!
Good call,  that makes sense.